pyproject.toml

```
@@ -26,7 +32,7 @@ python = "^3.7.2"
 email-validator = "^1.1.3"
 pydantic = "^1.9.0"
 python-dotenv = "^0.20.0"
-requests = "^2.27.1"
+requests = ">=2.14"
```

I think this would be better:
```suggestion
requests = "^2"
```
Which expands to
```
requests>=2.0.0,requests<3.0.0
```